.github/workflows/build.yml

```diff
@@ -126,7 +126,7 @@ jobs:
         path: /tmp/image
 
   build-stellar-soroban-rpc:
-    runs-on: ubuntu-latest
+    runs-on: ${{ inputs.core_build_runner_type }}
```

Comment:
@2opremio Why did we switch to using a 16x cpu runner for soroban-rpc?

Reply:
https://github.com/stellar/quickstart/pull/431#issuecomment-1443600020

Reply:
Thanks!